frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -258,6 +250,10 @@ class ManageAccounts extends Component<Props, State> {
           }
           return;
         }
+        const account = accounts.find(({ code }) => currentlyEditedAccount.code === code);
+        if (currentlyEditedAccount.active !== account?.active) {
+          this.toggleAccount(currentlyEditedAccount.code, currentlyEditedAccount.active);
+        }
```

Comment:
I don't like this, but now that the enable/disable account toggle is in the dialog, it should not fire right away but only after the user clicked the dialog update button.

Reply:
> this way we also call renameAccount when the name has not changed

true, as said I don't like it :)

reason it's messy it was 2 separate UI elements that are now combined into 1 dialog and updated together. 

One alternative could be to check first if either name or toggle changed and then only call the ones needed,.. but this make the function complicated. So I prefer to keep the logic for now. 

I change the function to updateAccount.